src/VsExtension/VsEvents/OnBuildPackageRestorer.cs

```diff
@@ -187,7 +194,7 @@ private void BuildEvents_OnBuildBegin(vsBuildScope scope, vsBuildAction Action)
             }
         }
 
-        private async Task RestoreBuildIntegratedProjectsAsync(IEnumerable<BuildIntegratedProjectSystem> buildEnabledProjects)
+        private async Task RestoreBuildIntegratedProjectsAsync(List<BuildIntegratedProjectSystem> buildEnabledProjects, bool forceRestore)
```

Comment:
break the long line, and add a comment that force restore is used when a rebuild is called